[4.0] Move the autoload file into the cache. Document in README (#28436)
joomla/joomla-cms: README.md

npm ci
```

**Things to be aware of when pulling:**

> Joomla creates a cache of its extensions namespaces into `JOOMLA_ROOT/administrator/cache/autoload_psr4.php`. If

Reviewer: ```suggestion
Joomla creates a cache of the namespaces of its extensions in `JOOMLA_ROOT/administrator/cache/autoload_psr4.php`. If
```

> Joomla creates a cache of it's extensions namespaces into `JOOMLA_ROOT/administrator/cache/autoload_psr4.php`. If

Reviewer 2: ```suggestion
Joomla creates a cache of its extensions namespaces into `JOOMLA_ROOT/administrator/cache/autoload_psr4.php`. If
```

> extensions are created, deleted or removed in git then this file needs to be re-created. You can simply delete the file

Reviewer: ```suggestion
extensions are created, deleted or removed in git then this file needs to be recreated. You can simply delete the file
```

Author: Thanks for both of these fixes!
